oauth2client/client.py

```diff
@@ -1533,15 +1549,16 @@ def verify_id_token(id_token, audience, http=None,
         CryptoUnavailableError: if no crypto library is available.
     """
     _require_crypto_or_die()
+
     if http is None:
-        http = transport.get_cached_http()
+        http = transport.get_http_object()
 
-    resp, content = transport.request(http, cert_uri)
-    if resp.status == http_client.OK:
-        certs = json.loads(_helpers._from_bytes(content))
-        return crypt.verify_signed_jwt_with_certs(id_token, certs, audience)
-    else:
-        raise VerifyJwtTokenError('Status code: {0}'.format(resp.status))
+    try:
+        certs = _get_certificates(http, cert_uri)
+    except http_client.HTTPException as e:
```

Comment:
Do you have a better recommendation? I've honestly never seen anyone else complain about `e` except in cases where there are multiple exceptions around.